test/e2e/redundancy/gateway_failover.go

```diff
@@ -41,12 +61,24 @@ func testEnginePodRestartScenario(f *subFramework.Framework) {
 	enginePod := f.AwaitSubmarinerEnginePod(framework.ClusterA)
 	By(fmt.Sprintf("Found submariner engine pod %q on %q", enginePod.Name, clusterAName))
 
+	By(fmt.Sprintf("Ensuring that the gateway reports as active %q", clusterAName))
+	activeGateways := f.GetGatewaysWithHAStatus(subv1.HAStatusActive)
+	Expect(activeGateways).To(HaveLen(1))
+
+	gw := activeGateways[0]
+	By(fmt.Sprintf("Ensuring that the gateway %q is reporting connections", gw.Name))
+	Expect(gw.Status.Connections).NotTo(BeEmpty())
+	Expect(gw.Status.Connections[0].Status).To(Equal(subv1.Connected))
```

Comment:
This could cause an intermittent failure if active not yet reporting connections - it seems this should call `AwaitGatewayFullyConnected`.

Reply:
Done.